CHANGELOG.md

```diff
@@ -7,6 +7,36 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 
 # Changelog
 
+## Forthcoming
+
+MotoROS2 is now built against `micro_ros_motoplus` version `20250207`.
+
+Breaking changes have been introduced in this version of MotoROS2, and failure to update and rebuild the workspace will lead to the inability to receive messages from MotoROS2 and/or crashes of client applications.
+
+**Breaking changes:**
+
+- Default joint names for single-group systems have been simplified (`joint_M`) ([#340](https://github.com/Yaskawa-Global/motoros2/pull/340))
+- Users who are updating from a lower version must remove the `motoros2_interfaces` packages from their Colcon workspaces (from the `src`, `build` and `install` spaces), and follow the installation instructions in the [Build and installation](https://github.com/yaskawa-global/motoros2_client_interface_dependencies#build-and-installation) section in the `README` of the new `yaskawa-global/motoros2_client_interface_dependencies` package.
+
+New functionality:
+
+- **Added support for ROS2 Jazzy Jalisco!** ([#337](https://github.com/Yaskawa-Global/motoros2/pull/337))
```

Comment:
Suggestion: make this an item / section by itself. It's not really new functionality as the other items in this section are: it's compatibility with a new version of ROS 2.

As such, this seems more like an announcement. Moving it right where we now only have "MotoROS2 is now built against [..]" would give it the attention it deserves, and it can also not be overlooked (ie: when scanning the changelog for "when did MotoROS2 get Jazzy support?").